### PR TITLE
Added Garshasp: The Monster Slayer

### PR DESCRIPTION
**Pull request by aposteriorist**
If you are adding or modifying an Auto Splitter, please fill out this checklist:
- [🗸] My Auto Splitter does not contain any malicious functionality and I'm entirely responsible for any damages myself. (Any kind of abuse in an Auto Splitter will result in an immediate ban.)
- [🗸] I am not replacing an existing Auto Splitter by a different author, or I have received permission from the author to replace the existing Auto Splitter.
- [🗸] The Auto Splitter has an open source license that allows anyone to fork and host it.

### CODE REVIEW

**Comment by just-ero** [comment]
I'm sorry, I'm not sure how to interpret those symbols in the check boxes. Please fill them out with a single X between the brackets: `[X]`.

File: LiveSplit.AutoSplitters.xml [hunk 1]
```diff
@@ -1,5 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <AutoSplitters>
+    <AutoSplitter>
+        <Games>
+            <Game>Garshasp: The Monster Slayer</Game>
+        </Games>
+        <URLs>
+            <URL>https://raw.githubusercontent.com/aposteriorist/PeachPit.LiveSplit/main/LiveSplit.GarshaspTheMonsterSlayer.asl</url>
```
Comment (just-ero) [inline LiveSplit.AutoSplitters.xml:8]:
```suggestion
            <URL>https://raw.githubusercontent.com/aposteriorist/PeachPit.LiveSplit/main/LiveSplit.GarshaspTheMonsterSlayer.asl</URL>
```

**Comment by just-ero** [comment]
@aposteriorist Please review and apply the requested changes. Your additions aren't passing tests.

**Comment by aposteriorist** [comment]
Sorry, I hadn't realized that I needed to approve the edit on my end.

**Pull request merged**